simulariumio/data_objects/trajectory_data.py

```diff
@@ -29,9 +29,9 @@ def __init__(
         self,
         meta_data: MetaData,
         agent_data: AgentData,
-        time_units: UnitData = UnitData("s"),
-        spatial_units: UnitData = UnitData("m"),
-        plots: List[Dict[str, Any]] = [],
+        time_units: UnitData = None,
+        spatial_units: UnitData = None,
+        plots: List[Dict[str, Any]] = None,
```

Comment:
is this change unrelated to the display data?

Reply:
yeah this is part of "make default parameters for collections None" to prevent future bugs, if they default to e.g. `[]` there can be weirdness when using them.